bin/arm64_env.sh

```diff
@@ -17,4 +17,11 @@ export supportRulesengine=nexus3.edgexfoundry.org:10004/docker-support-rulesengi
 export exportClient=nexus3.edgexfoundry.org:10004/docker-export-client-go-arm64:1.0.0
 export exportDistro=nexus3.edgexfoundry.org:10004/docker-export-distro-go-arm64:1.0.0
 export deviceVirtual=nexus3.edgexfoundry.org:10004/docker-device-virtual-arm64:0.6.0
+
+export vault=nexus3.edgexfoundry.org:10004/docker-edgex-vault-arm64:0.3.0
+export vaultWorker=nexus3.edgexfoundry.org:10004/docker-edgex-vault-worker-arm64:0.3.0
```

Comment:
this should be `export vaultWorker=nexus3.edgexfoundry.org:10004/docker-edgex-vault-worker-go-arm64:0.3.0`